### ISSUE TITLE
Question: How to load jquery-ui on scripted dashboards

### ISSUE THREAD

**Issue by tlopo** [grafana/grafana#2182]
Hi There,

First of all I love grafana, scripted dashboard rocks so I would like to say a big thank you to all contributors.

I am not a web dev, but a very curious sysadmin.

I was able to create nice dropdowns using the  result of 'list series' influxdb query and it all worked very well.

Now I would like to use multiselect jquery plugin which requires jquery-ui. I can see the network calls  and the  file is being loaded but it's just not working, on console 'jQuery.ui' returns undefined.

I wonder if there any special step to load jquery plugins in grafana.

Thanks,
Tiago

**Comment by torkelo** [comment]
Not sure what you trying to do. Drop down in scripted dashboard,  you mean the template variable value drop down? 

Scripted dashboards do not support UI libs, only js libs that help you build a dashboard model

**Comment by tlopo** [comment]
A scripted dashboard must return a valid dashboard or a function, I am returning a dashboard with a text/html panel, in that panel I've added a script tag which creates the drop-downs for me. Then I can select the metrics I want and when I am done it redirects to the same scripted dashboard but with data in the query string so I can add panels to  the dashboard. It's working fine, but I wrote it in plain javascript  and it's not very reusable. I want to add more drop-downs  and jquery-ui would make it a lot easier.

Doing this way, even when save the dashboard  the drop-downs still work. 

But anyway, I think  there is no issue because my do nothing jquery plugin below works fine:

```
(function ( $ ) {
        $.fn.donothing = function (){
                return this;
        }
}( jQuery ));
```  

there may be something wrong with the jquery-ui.js I am trying to load.
